newt_build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -x
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+# This script should be used from mynewt-newt source root directory
+
+mkdir -p $GOPATH/bin
+export PATH=$GOPATH/bin:$PATH
+
+if [[ ${TRAVIS_OS_NAME} = "windows" ]]; then
+    echo 'Building newt for Windows'
+    pushd newt/ || exit
+    go build
+    mv newt.exe newt
```

Comment:
Those steps would also work for Linux and macos, apart from the `mv`

Reply:
I know but I thought that by using build.sh script we will also test it, so that's why I kept it on Linux and MacOS. I'm fine with changing it though.

Reply:
It's OK, eventually would be better to just get rid of `build.sh` and use `go build` in all tooling (mainly release packages generation I suppose).